addons/website/views/snippets.xml

```diff
@@ -760,9 +709,63 @@
                <li data-styling="shadow"><a href="#">Shadows</a></li>
            </ul>
         </li>
-       <li class="separator"></li>
-       <li data-images_add="true"><a href="#"><i class="fa fa-plus-circle"/>Add images</a></li>
-       <li data-images_rm="true"><a href="#"><i class="fa fa-trash"/>Remove all images</a></li>
+    </div>
+
+    <div data-js='gallery' data-selector=".o_gallery">
```

Comment:
to move the 'Add Images' and 'Remove All Images' options in the top of customize menu